### PR TITLE
Update ICPC Achievement

### PR DESCRIPTION
**Pull request by pranshukas**
Added the ICPC Achievements in carrers.yml

### CODE REVIEW

**Comment by pranshukas** [comment]
Please Review and revert in case any changes need to be made !

File: _data/index/careers.yml [hunk 1]
```diff
@@ -1,5 +1,19 @@
...
 # if you don't have language feature(language.yml is empty), ignore "i18n" items
+- name:
+    detail: ACM ICPC 2020
+    i18n: company_a
+  desc:
+    detail: Team rng(weed) (Jatin Garg, Anshu Garg, Yash Goyal) topped Kanpur-Mathura Regionals and have chance to Represent IIT BHU in World Finals. <br><br> 
+            23 Teams from the College qualified for ICPC Amritapuri Regionals. 
+    i18n: company_a_desc
+  date:
+    detail: Sept. 2021
```
Comment (nishantwrp) [inline _data/index/careers.yml]: Did it happen in Sept 2021?

Comment (pranshukas) [reply _data/index/careers.yml]: It happened in Month of Aug. Updating !

**Review by nishantwrp — APPROVED** [review]
Thanks for the PR @pranshukas!

**Pull request merged**